NGitLab.Mock.Tests/IssuesMockTests.cs

```diff
@@ -55,5 +55,21 @@ public void Test_issues_assignee_not_throwing_when_assignees_is_null()
             var client = server.CreateClient();
             Assert.DoesNotThrow(() => client.Issues.Get(new IssueQuery { Scope = "assigned_to_me" }).ToArray());
         }
+
+        [Test]
+        public void Test_issue_by_id_can_be_found()
+        {
+            using var server = new GitLabConfig()
+                .WithUser("user", isDefault: true, isAdmin: true)
+                .WithProject("Test", configure: project => project
+                    .WithIssue("Issue title", author: "user", id: 5))
```

Comment:
Can you maybe use the WithIssue with the configure action overload?

To see if you can retrieve the internal Id (`10001` looks like it's out of nowhere :))

Reply:
I thought it would have been easier to get it, indeed it seems quite hidden :).